packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk/PaymentSheetFragment.kt

```diff
@@ -145,7 +145,9 @@ class PaymentSheetFragment(
         putBoolean("shouldSavePaymentMethod", shouldSavePaymentMethod)
       }
 
-      stripeSdkModule.sendEvent(context, "onConfirmHandlerCallback", params)
+      context.currentActivity?.runOnUiThread {
```

Comment:
I am not convinced about this change since it will block the UI. Also this change needs to be ported to https://github.com/stripe/stripe-react-native since we copy the sdk 1 by 1